source/includes/opts/loadlevel-warning.rst

```diff
@@ -1,4 +1,8 @@
 .. warning::
 
    Setting ``loadLevel`` higher than the default of ``3`` may negatively
-   impact the destination cluster performance.
\ No newline at end of file
+   impact the destination cluster performance.
+
+   If read bottlenecks exist on the source cluster or if write
+   bottlenecks exist on the destination cluster, decreasing
+   ``loadLevel`` may improve performance. 
```

Comment:
[suggestion]

Consider updating to "might improve performance". 

Source: https://www.mongodb.com/docs/meta/style-guide/writing/use-effective-verbs/#use-modal-verbs-accurately

Reply:
Done!